### PR TITLE
feat: added new command to swipe right and left

### PR DESCRIPTION
**Pull request by amuthansakthivel**
## Proposed changes
We introduced the scrollTo element in the previous versions and thought the left and right directions were unnecessary. 

But there have been consistent requests to have the option to search for elements by swiping right and left.

I could have added to the existing scroll command but the readability would be awkward. Hence created a new swipe command which is what it needs to be called. 

Sample usage is updated in the readme.md

$(element).swipeTo().click();

## Checklist
- [x] Checkstyle and unit tests are passed locally with my changes by running `gradlew check chrome_headless firefox_headless` command
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: modules/appium/src/main/java/com/codeborne/selenide/appium/commands/SelenideAppiumCommands.java [hunk 1]
```diff
@@ -16,5 +16,6 @@ public SelenideAppiumCommands() {
     add("hideKeyboard", new HideKeyboard());
     add("scrollTo", new AppiumScrollTo());
     add("scroll", new AppiumScrollTo());
+    add("swipe", new AppiumSwipeTo());
```
Comment (OleksandrOwl) [inline modules/appium/src/main/java/com/codeborne/selenide/appium/commands/SelenideAppiumCommands.java:19]: Чи потрібно сюди додати ще `add("swipeTo", new AppiumSwipeTo());` ?

Comment (amuthansakthivel) [reply modules/appium/src/main/java/com/codeborne/selenide/appium/commands/SelenideAppiumCommands.java:19]: Fixed it. Good catch!

Comment (asolntsev) [reply modules/appium/src/main/java/com/codeborne/selenide/appium/commands/SelenideAppiumCommands.java:19]: @amuthan-sakthivel You added only "swipe" command, but don't you also need to add "swipeTo" command?

**Review by asolntsev — CHANGES_REQUESTED** [review]

File: modules/appium/src/main/java/com/codeborne/selenide/appium/commands/SelenideAppiumCommands.java (hunk 1 above)
Comment (amuthansakthivel) [reply modules/appium/src/main/java/com/codeborne/selenide/appium/commands/SelenideAppiumCommands.java:19]: @asolntsev - SwipeTo is already there

image

**Review by asolntsev — APPROVED** [review]

**Comment by asolntsev** [comment]
@amuthansakthivel The build is failing. Can you fix it?

> Task :modules:appium:checkstyleMain
> Error: eckstyle] [ERROR] > AppiumSwipeOptions.java:3:65: Using the '.*' form of import should be avoided - com.codeborne.selenide.appium.AppiumSwipeDirection.*. [AvoidStarImport]

**Comment by amuthansakthivel** [comment]
Pushed the fix @asolntsev

**Review by asolntsev — APPROVED** [review]

**Pull request merged**